cli/tests/helm-tests/helm_test.go

```diff
@@ -86,10 +86,18 @@ func testHelmClientInstallOrUpgradeChart() {
 		log.Fatalf("Error installing tobs secrets %v:", err)
 	}
 
+	// download the dependent charts
+	out := exec.Command("helm", "dep", "up", LATEST_CHART)
+	output, err := out.CombinedOutput()
+	if err != nil {
+		fmt.Println(string(output))
+		log.Fatal(err)
+	}
+
 	// Define the chart to be installed
 	chartSpec := helm.ChartSpec{
 		ReleaseName:     DEFAULT_TOBS_NAME,
-		ChartName:       CHART_NAME,
+		ChartName:       LATEST_CHART,
```

Comment:
The helm tests are intended to test the functionality of helm funcs. Here we test helm using the last released version that is published to the helm charts repo. 

As the last version fails to work with the latest K8s versions i.e. `1.22` and `1.23`. Changed the tests temporarily to point to the latest chart that is part of the gh repo.

Once we publish the helm-chart to charts repo (s3 bucket). We should revert back this change. I know this is a bit messy. But I don't see any other workaround for this. The helm tests are intended to test with an already released chart to download and install it tobs.

Reply:
I would say let's test what we have in s3 bucket as this is what users also see and only add a step to promscale to publish chart instead of having to refactor this code again in short future.

Reply:
This is about the helm chart in tobs gh repo. We bumped otel operator and haven't published it yet. So this helped to pass CI and test is there anything breaking on helm side. 

However, after publishing tobs helm chart I revert this.